python/sglang/srt/managers/schedule_batch.py

```diff
@@ -197,16 +197,14 @@ def check_finished(self):
             return
 
         if len(self.output_ids) >= self.sampling_params.max_new_tokens:
-            self.finished_reason = FINISH_LENGTH(len(self.output_ids))
+            self.finished_reason = "length"
             return
 
         if (
             self.output_ids[-1] == self.tokenizer.eos_token_id
             and not self.sampling_params.ignore_eos
         ):
-            self.finished_reason = FINISH_MATCHED_TOKEN(
-                matched=self.tokenizer.eos_token_id
-            )
+            self.finished_reason = "stop"
```

Comment:
Do one of the following:
1. fully deprecated the custom finish reason
2. still use custom finish reason in srt but convert it to OpenAI format in `adapter.py`

Reply:
fixed